src/routes/auth/login/SocialAuth.jsx

```diff
@@ -44,17 +49,18 @@ const SocialAuth = ({ socket }) => {
   };
 
   const activateBuyerPrivileges = () => {
-    disableButton(true);
+    setIsButtonDisabled(true);
+    popUpWindow = openPopup();
     checkPopup();
-    pop = openPopup();
+    // popUpWindow.close()
```

Comment:
Do not leave commented code when submitting a PR

Reply:
Sorry I missed out on this